cumulusci/core/config/SfdxOrgConfig.py

```diff
@@ -59,6 +60,7 @@ def sfdx_info(self):
             "access_token": org_info["result"]["accessToken"],
             "org_id": org_id,
             "username": org_info["result"]["username"],
+            "access_token_cache": {},
```

Comment:
Can we put this on a separate attribute (self._access_token_cache) please? Two reasons:
1. sfdx_info currently only contains information from the `force:org:display` command for the admin user, and I'd rather keep info related to other users separate.
2. sfdx_info is added to self.config below on line 67, which means it will get stored on the filesystem if the org config is saved to the keychain. I don't think we want to do this. Access tokens expire after a period of time, but you haven't implemented anything to invalidate the cache. So it'll probably work best if we get a new access token each time cci is run.

Reply:
I'm almost inclined to say that we shouldn't cache it at all. It's not something that we're likely to need to get multiple times for the same user very often, and when we do, it's safest to call out to sfdx each time to make sure that we have a token that hasn't expired.

Reply:
If you want it in a separate attribute, I can do that. I originally had it that way, but figured combining with the others made more sense. 

As for the need for a cache at all, I was concerned that a test might need to open a browser for a second user multiple times, and I didn't want to add what could be minutes to an already long test run. I see your point about an expired token, though we cache the token for the default user and don't seem to have a problem with it expiring.

Reply:
I guess I can remove the cache and see if anyone complains about performance. We can always add it back in later.

Reply:
Yeah, I'd prefer to wait and add the cache only if it's needed.